frontend/src/app/data-entry/agent-form/agent-historical-person-form/agent-historical-person-form.component.html

```diff
@@ -0,0 +1,17 @@
+<fieldset *ngIf="combined$ | async as combined">
+    <legend>
+        {{ combined.isGroup ? "Historical persons" : "Historical person" }}
+    </legend>
+    <p class="form-text">
+        {{
+            combined.isGroup
+                ? "Pick one or more historical persons that are described by this agent in this source."
+                : "Pick an historical person that is described by this agent in this source."
```

Comment:
Perhaps it would be clearer to say "historical person(s) that this agents represents". (So this is not historical figures that the agent diegetically describes.)